### DISCUSSION TITLE
如何修改pcl2打开如mcbbs等网站时用浏览器？

### DISCUSSION THREAD

**Discussion by lifehxd** [Meloong-Git/PCL#2040] [提问] [answered]
如何修改pcl2打开如mcbbs等网站时用浏览器？

**Comment by WForst-Breeze** [comment]
如果您指的是使用自定义主页来实现的话，可以添加一个 local:MyButton，示例如下：
```xaml
<local:MyButton Width="140" Height="35" HorizontalAlignment="Left" Padding="13,0,13,0"
                Text="打开 MCBBS" EventType="打开网页" EventData="https://www.mcbbs.net/" />
```

有关更多自定义主页内容你可以前往 PCL2 内置帮助库 **个性化-自定义主页与帮助 XAML 格式** 中查看

---

如果您想修改 PCL2 打开连接时使用的浏览器，请前往 Windows 设置，修改您的默认浏览器，具体方法见下方朋友给出的描述

**Reply by lifehxd** [reply]
已解决，谢谢了

**Comment by Chiloven945** [comment, marked as answer]
打开设置，选择`应用>默认应用`，找到你想要使用的浏览器，进入页面，点击上方位于`将xxx设为默认浏览器`的`设为默认值`按钮

**Reply by lifehxd** [reply]
已解决，谢谢了
